changelog.d/0-release-notes/cargohold-domain

```diff
@@ -0,0 +1 @@
+This release introduces a mandatory `federationDomain` configuration setting to cargohold. Please update your `values/wire-server/values.yaml` to set `cargohold.settings.federationDomain` to the same value as the corresponding option in galley (and brig).
```

Comment:
this doesn't match the title of the PR.

Reply:
True, but it would be somewhat painful to split this into two PR, since the ZAuth servant combinators have integration with the `Qualified` stuff that needs knowledge of the federation domain, which we'll need in cargohold later anyway. It would just be more pointless work to do this in two steps.

Can I just expand the PR description to include this?

Reply:
I updated the title and description. Ok to merge?